### PR TITLE
UMD and register on npm

### PR DESCRIPTION
**Pull request by shelldandy**

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,5 +1,9 @@
 #jQuery.isOnScreen
 
+Original plugin: https://github.com/moagrius/isOnScreen
+
+I just cloned it and wrapped for UMD
```
Comment (moagrius) [inline README.md:5]: please remove these additions.  you can add something at the end in 3rd person like "updated for UMD by Miguel Palau <mpalauzarza@gmail.com>"

File: index.js [hunk 2]
```diff
@@ -1,16 +1,13 @@
...
+(function (factory) {
+  'use strict'
+  if (typeof define === 'function' && define.amd) {
+    define(['jquery'], factory)
+  } else if (typeof exports !== 'undefined') {
+    module.exports = factory(require('jquery'))
+  } else {
+    factory(jQuery)
+  }
+}(function ($) {
```
Comment (moagrius) [inline index.js:10]: please add the jsdocs back in

File: jquery.isonscreen.min.js [hunk 3]
```diff
@@ -1 +0,0 @@
-!function(a){a.fn.isOnScreen=function(b){var c=this.outerHeight(),d=this.outerWidth();if(!d||!c)return!1;var e=a(window),f={top:e.scrollTop(),left:e.scrollLeft()};f.right=f.left+e.width(),f.bottom=f.top+e.height();var g=this.offset();g.right=g.left+d,g.bottom=g.top+c;var h={top:f.bottom-g.top,left:f.right-g.left,bottom:g.bottom-f.top,right:g.right-f.left};return"function"==typeof b?b.call(this,h):h.top>0&&h.left>0&&h.right>0&&h.bottom>0}}(jQuery);
```
Comment (moagrius) [inline jquery.isonscreen.min.js:1]: can you add a minified version?

File: package.json [hunk 4]
```diff
@@ -0,0 +1,12 @@
+{
+  "name": "jquery-on-screen",
+  "version": "1.0.0",
+  "description": "hitesh",
```
Comment (moagrius) [inline package.json:4]: what is "hitesh"?

File: package.json [hunk 5]
```diff
@@ -0,0 +1,12 @@
...
+  "main": "index.js",
+  "repository": "git@github.com:mike3run/isOnScreen.git",
+  "author": "Miguel Palau <mpalauzarza@gmail.com>",
+  "license": "MIT",
+  "peerDependencies": {
+    "jquery": ">= 1.6.2"
```
Comment (moagrius) [inline package.json:10]: i think it'll work on earlier versions?

**Review by moagrius — CHANGES_REQUESTED** [review]

**Comment by shelldandy** [comment]
I honestly didn't expect you to reply so fast haha thanks!

My friend Hitesh uses this package but in a "vanilla 1998" way but since he started trying webpack because of me he didn't get this to work, so I just wrapped your code on UMD format and called it a day.

I'll try to apply your requested changes on the weekend and come back here

Thanks!

**Pull request closed without merging**